Jenkinsfile

```diff
@@ -19,7 +26,7 @@ elifePipeline {
     stage 'Project tests', {
         lock('pattern-library--ci') {
             builderDeployRevision 'pattern-library--ci', commit
-            builderProjectTests 'pattern-library--ci', '/srv/pattern-library'
+            builderSmokeTests 'pattern-library--ci', '/srv/pattern-library'
```

Comment:
we don't need to run project tests here since they are run by the `ci` container, only smoke tests remain